lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,316 @@
+from django.core.management.base import BaseCommand, CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from django.test import TestCase
+
+from .. import rename_user
+
+
+class TestRenameUser(TestCase):
+    NEW_USERNAME = "targetName"
+
+
```

Comment:
too many whitespace lines

you probably just want to do paver run_quality and fix all the reported issues